components/tidb_query_common/src/string.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
`tidb_query_shared_expr` is the more appropriate location to place these. You can update the branch then see something overlapped already exists there.

Reply:
@zhongzc Comment addressed. PTAL~